atomic_reactor/utils/operator.py

```diff
@@ -210,7 +361,9 @@ def replace_pullspecs_everywhere(self, replacement_pullspecs):
         :param replacement_pullspecs: mapping of pullspec -> replacment
         """
         for k in self.data:
-            self._replace_pullspecs_everywhere(self.data, k, replacement_pullspecs)
+            self._replace_pullspecs_not_in_annotations(self.data, k, replacement_pullspecs)
+        for annotation in self._annotation_pullspecs() + self._guess_annotation_pullspecs():
```

Comment:
do we still need "self._annotation_pullspecs()" ???
self._guess_annotation_pullspecs() will find them anyway